files/build_templates/init_cfg.json.j2

```diff
@@ -1,6 +1,7 @@
 {
     "DEVICE_METADATA": {
         "localhost": {
+            "buffer_model": {% if default_buffer_model == "dynamic" %}"dynamic"{% else %}"traditional"{% endif %},
```

Comment:
>traditional [](start = 90, length = 11)

dynamic vs static?

Reply:
`dynamic` / `traditional` is the term we already agreed on, isn't it?